Tensor construction codemod(ResizeLike) - 2/3

jerryzh168 opened the pull request:
Summary:
Codemod generated with clangr shard mode, 25 files per diff,
motivation: https://github.com/pytorch/pytorch/pull/12407

Differential Revision: D13629047

smessmer commented on caffe2/operators/integral_image_op.cu:
```diff
@@ -165,10 +165,10 @@ bool IntegralImageGradientOp<float, CUDAContext>::RunOnDevice() {
   auto& dY = Input(1); // Gradient of net w.r.t. output of "forward" op
                        // (aka "gradOutput")
-  auto* dX = Output(0); // Gradient of net w.r.t. input to
+   // Gradient of net w.r.t. input to
```
This comment should probably move too

smessmer commented on caffe2/operators/roi_align_gradient_op.cu:
```diff
@@ -193,10 +193,10 @@ bool RoIAlignGradientOp<float, CUDAContext>::RunOnDevice() {
   auto& dY = Input(2); // Gradient of net w.r.t. output of "forward" op
                        // (aka "gradOutput")
-  auto* dX = Output(0); // Gradient of net w.r.t. input to
+   // Gradient of net w.r.t. input to
```
and here too

smessmer commented on caffe2/operators/roi_pool_op.cu:
```diff
@@ -173,10 +173,10 @@ bool RoIPoolGradientOp<float, CUDAContext>::RunOnDevice() {
   auto& dY = Input(3); // Gradient of net w.r.t. output of "forward" op
   // (aka "gradOutput")
-  auto* dX = Output(0); // Gradient of net w.r.t. input to "forward" op
+   // Gradient of net w.r.t. input to "forward" op
```
and here

smessmer commented on caffe2/operators/softmax_ops.cu:
```diff
@@ -284,14 +284,14 @@ template<>
   auto& X = Input(0);  // Logits
   auto& T = Input(1);  // Labels / targets
-  auto* P = Output(0); // Probabilities from softmax
+   // Probabilities from softmax
```
here

smessmer commented on caffe2/operators/softmax_ops.cu:
```diff
@@ -391,13 +391,13 @@ template <>
   auto& X = Input(0); // Logits
   auto& T = Input(1); // Labels / targets
-  auto* P = Output(0); // Probabilities from softmax
+   // Probabilities from softmax
```
here

jerryzh168 committed:
Tensor construction codemod(ResizeLike) - 2/3 (#15940)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/15940

Codemod generated with clangr shard mode, 25 files per diff,
motivation: https://github.com/pytorch/pytorch/pull/12407

Reviewed By: smessmer

Differential Revision: D13629047

fbshipit-source-id: 8316f9f85d7de48d5657d80afa28b16e11b35d5f
